delayed_job/monitor_delayed_jobs.rb

```diff
@@ -10,14 +10,13 @@ class MonitorDelayedJobs < Scout::Plugin
     default: production
   queue_name:
     name: Queue Name
```

Comment:
Not strictly necessary to quote strings with spaces, just makes it clearer. Will do them all though.